Resources/Prototypes/Reagents/Consumable/Food/condiments.yml

```diff
@@ -155,3 +155,19 @@
       # eating salt on its own kinda sucks, kids
       - !type:SatiateThirst
         factor: -0.5
+      - !type:HealthChange
+        conditions:
+        - !type:ReagentThreshold
+          min: 1
+        - !type:OrganType
+          type: HeadSlime # Salt Kills you if your a HeadSlime
+        damage:
+          types:
+            Poison: 40
```

Comment:
40 damage per unit seems a bit extreme, maybe just 10 or even 5? I can see the CMO going around with their hypospray oneshotting every headslime they come across.

Reply:
On second thought, I can see that the whole point is that salt is incredibly deadly, so 40 might be fine. Something like 19 damage might be more reasonable though.

Reply:
This damage is per second, not per unit. This gives salt 80 damage per unit. I’d half it (at least) because as-is 2u of salt will kill one in 3 seconds which is just absurd.